contributing.md

```diff
@@ -42,6 +42,8 @@ Before starting major work, you should touch base with the maintainers of Featur
 
 * You will need to install Spark, Scala, GraphViz, and Pandoc to run all unit tests & build docs:
 
+  > If you do not install Spark/Scala, you can still run the unit tests (the Spark tests will be skipped).
+
```

Comment:
Should we add another note stating that Pandoc is only needed for building the docs?

Reply:
Done